.studio/common

```diff
@@ -251,8 +251,10 @@ function verify_component() {
 }
 
 function display_available_components() {
-  log_line "Available components:"
-  ls -1 /src/components | awk '{print "* "$1}'
+  log_line "Available targets:"
+  echo "  * api"
+  echo "  * config"
+  ls -1 /src/components/*/scripts/grpc.sh | sed -r 's/\/src\/components\/(.*)\/scripts\/grpc.sh/  * \1/g'
```

Comment:
[nit] This should do the trick, too 😉 `-F/` sets the field separator to `/`, and with that, `$4` would be the dir below `/src/components/` (because `$1` is `""`)
```suggestion
  ls -1 /src/components/*/scripts/grpc.sh | awk -F/ '{ print "  * "$4 }'
```
(⚠️ untested)

Reply:
Bloody 'ell Stephan, that works:
```
[3][default:/src:0]# ls -1 /src/components/*/scripts/grpc.sh | awk -F/ '{ print "  * "$4 }'
  * automate-deployment
  * automate-gateway
  * automate-grpc
  * compliance-service
  * data-lifecycle-service
  * nodemanager-service
```

I'll take it, thanks!